### PR TITLE
Use null-aware operators in package:sky

### PR DESCRIPTION
**Pull request by abarth**

### CODE REVIEW

**Comment by abarth** [comment]
@Hixie @HansMuller

**Comment by chinmaygarde** [comment]
Whoa. Dart has elvis operators now? lgtm

**Comment by abarth** [comment]
Elvis?  yeppers.

File: sky/packages/sky/lib/src/widgets/framework.dart [hunk 1]
```diff
@@ -1259,7 +1259,7 @@ abstract class OneChildRenderObjectWrapper extends RenderObjectWrapper {
 
   void syncRenderObject(RenderObjectWrapper old) {
```
Comment (Hixie) [inline sky/packages/sky/lib/src/widgets/framework.dart]: you should change the signature of the method to take a OneChildRenderObjectWrapper and drop the cast on the line you modified

**Comment by Hixie** [comment]
LGTM

**Pull request merged**